Unity package.json is missing dll references in v2.1.165 if dll are not auto referenced

#### Bug description
```
Library\PackageCache\com.neuecc.messagepack@7d06c13e72\ExtensionResult.cs(30,16): error CS0122: 'ReadOnlySequence<T>' is inaccessible due to its protection level
Library\PackageCache\com.neuecc.messagepack@7d06c13e72\BufferWriter.cs(90,16): error CS0122: 'Span<T>' is inaccessible due to its protection level
Library\PackageCache\com.neuecc.messagepack@7d06c13e72\BufferWriter.cs(100,18): error CS0122: 'IBufferWriter<T>' is inaccessible due to its protection level
Library\PackageCache\com.neuecc.messagepack@7d06c13e72\ExtensionResult.cs(16,48): error CS0122: 'Memory<T>' is inaccessible due to its protection level
Library\PackageCache\com.neuecc.messagepack@7d06c13e72\ExtensionResult.cs(22,48): error CS0122: 'ReadOnlySequence<T>' is inaccessible due to its protection level
Library\PackageCache\com.neuecc.messagepack@7d06c13e72\BufferWriter.cs(104,16): error CS0122: 'Span<T>' is inaccessible due to its protection level
Library\PackageCache\com.neuecc.messagepack@7d06c13e72\BufferWriter.cs(160,27): error CS0122: 'ReadOnlySpan<T>' is inaccessible due to its protection level
Library\PackageCache\com.neuecc.messagepack@7d06c13e72\BufferWriter.cs(191,49): error CS0122: 'ReadOnlySpan<T>' is inaccessible due to its protection level
Library\PackageCache\com.neuecc.messagepack@7d06c13e72\BufferWriter.cs(228,39): error CS0122: 'ReadOnlySpan<T>' is inaccessible due to its protection level
Library\PackageCache\com.neuecc.messagepack@7d06c13e72\BufferWriter.cs(55,29): error CS0122: 'IBufferWriter<T>' is inaccessible due to its protection level
Library\PackageCache\com.neuecc.messagepack@7d06c13e72\BufferWriter.cs(22,17): error CS0122: 'IBufferWriter<T>' is inaccessible due to its protection level
Library\PackageCache\com.neuecc.messagepack@7d06c13e72\BufferWriter.cs(28,17): error CS0122: 'Span<T>' is inaccessible due to its protection level
Library\PackageCache\com.neuecc.messagepack@7d06c13e72\Internal\Sequence`1.cs(28,34): error CS0122: 'IBufferWriter<T>' is inaccessible due to its protection level
Library\PackageCache\com.neuecc.messagepack@7d06c13e72\Internal\Sequence`1.cs(97,16): error CS0122: 'ReadOnlySequence<T>' is inaccessible due to its protection level
Library\PackageCache\com.neuecc.messagepack@7d06c13e72\Internal\AutomataDictionary.cs(53,36): error CS0122: 'ReadOnlySequence<T>' is inaccessible due to its protection level
Library\PackageCache\com.neuecc.messagepack@7d06c13e72\Internal\AutomataDictionary.cs(55,33): error CS0122: 'ReadOnlySpan<T>' is inaccessible due to its protection level
Library\PackageCache\com.neuecc.messagepack@7d06c13e72\Internal\AutomataDictionary.cs(207,48): error CS0122: 'ReadOnlySpan<T>' is inaccessible due to its protection level
Library\PackageCache\com.neuecc.messagepack@7d06c13e72\Internal\AutomataDictionary.cs(422,40): error CS0122: 'ReadOnlySpan<T>' is inaccessible due to its protection level
Library\PackageCache\com.neuecc.messagepack@7d06c13e72\Internal\ByteArrayStringHashTable.cs(83,36): error CS0122: 'ReadOnlySequence<T>' is inaccessible due to its protection level
Library\PackageCache\com.neuecc.messagepack@7d06c13e72\Internal\ByteArrayStringHashTable.cs(86,33): error CS0122: 'ReadOnlySpan<T>' is inaccessible due to its protection level
Library\PackageCache\com.neuecc.messagepack@7d06c13e72\Internal\ByteArrayStringHashTable.cs(108,38): error CS0122: 'ReadOnlySpan<T>' is inaccessible due to its protection level
Library\PackageCache\com.neuecc.messagepack@7d06c13e72\Internal\ByteArrayStringHashTable.cs(127,51): error CS0122: 'ReadOnlySpan<T>' is inaccessible due to its protection level
Library\PackageCache\com.neuecc.messagepack@7d06c13e72\Internal\CodeGenHelpers.cs(60,65): error CS0122: 'ReadOnlySequence<T>' is inaccessible due to its protection level
Library\PackageCache\com.neuecc.messagepack@7d06c13e72\Internal\CodeGenHelpers.cs(60,23): error CS0122: 'ReadOnlySpan<T>' is inaccessible due to its protection level
Library\PackageCache\com.neuecc.messagepack@7d06c13e72\Internal\CodeGenHelpers.cs(82,66): error CS0122: 'ReadOnlySequence<T>' is inaccessible due to its protection level
Library\PackageCache\com.neuecc.messagepack@7d06c13e72\Internal\CodeGenHelpers.cs(82,24): error CS0122: 'ReadOnlySpan<T>' is inaccessible due to its protection level
Library\PackageCache\com.neuecc.messagepack@7d06c13e72\Internal\CodeGenHelpers.cs(70,23): error CS0122: 'ReadOnlySpan<T>' is inaccessible due to its protection level
Library\PackageCache\com.neuecc.messagepack@7d06c13e72\Internal\CodeGenHelpers.cs(80,62): error CS0122: 'ReadOnlySequence<T>' is inaccessible due to its protection level
Library\PackageCache\com.neuecc.messagepack@7d06c13e72\LZ4\LZ4Codec.Unsafe.cs(42,41): error CS0122: 'ReadOnlySpan<T>' is inaccessible due to its protection level
Library\PackageCache\com.neuecc.messagepack@7d06c13e72\LZ4\LZ4Codec.Unsafe.cs(42,67): error CS0122: 'Span<T>' is inaccessible due to its protection level
Library\PackageCache\com.neuecc.messagepack@7d06c13e72\LZ4\LZ4Codec.Unsafe.cs(89,41): error CS0122: 'ReadOnlySpan<T>' is inaccessible due to its protection level
Library\PackageCache\com.neuecc.messagepack@7d06c13e72\LZ4\LZ4Codec.Unsafe.cs(89,67): error CS0122: 'Span<T>' is inaccessible due to its protection level
Library\PackageCache\com.neuecc.messagepack@7d06c13e72\Internal\FarmHash.cs(19,42): error CS0122: 'ReadOnlySpan<T>' is inaccessible due to its protection level
Library\PackageCache\com.neuecc.messagepack@7d06c13e72\Internal\FarmHash.cs(80,50): error CS0122: 'ReadOnlySpan<T>' is inaccessible due to its protection level
Library\PackageCache\com.neuecc.messagepack@7d06c13e72\Internal\FarmHash.cs(203,43): error CS0122: 'ReadOnlySpan<T>' is inaccessible due to its protection level
Library\PackageCache\com.neuecc.messagepack@7d06c13e72\Internal\GuidBits.cs(185,35): error CS0122: 'ReadOnlySpan<T>' is inaccessible due to its protection level
Library\PackageCache\com.neuecc.messagepack@7d06c13e72\Internal\GuidBits.cs(312,27): error CS0122: 'Span<T>' is inaccessible due to its protection level
Library\PackageCache\com.neuecc.messagepack@7d06c13e72\Internal\GuidBits.cs(61,25): error CS0122: 'ReadOnlySpan<T>' is inaccessible due to its protection level
Library\PackageCache\com.neuecc.messagepack@7d06c13e72\MessagePackReader.cs(69,16): error CS0122: 'ReadOnlySequence<T>' is inaccessible due to its protection level
Library\PackageCache\com.neuecc.messagepack@7d06c13e72\MessagePackReader.cs(74,16): error CS0122: 'SequencePosition' is inaccessible due to its protection level
Library\PackageCache\com.neuecc.messagepack@7d06c13e72\MessagePackReader.cs(119,43): error CS0122: 'ReadOnlySequence<T>' is inaccessible due to its protection level
Library\PackageCache\com.neuecc.messagepack@7d06c13e72\MessagePackReader.cs(268,16): error CS0122: 'ReadOnlySequence<T>' is inaccessible due to its protection level
Library\PackageCache\com.neuecc.messagepack@7d06c13e72\MessagePackReader.cs(289,16): error CS0122: 'ReadOnlySequence<T>' is inaccessible due to its protection level
Library\PackageCache\com.neuecc.messagepack@7d06c13e72\MessagePackReader.cs(669,16): error CS0122: 'ReadOnlySequence<T>' is inaccessible due to its protection level
Library\PackageCache\com.neuecc.messagepack@7d06c13e72\MessagePackReader.cs(694,16): error CS0122: 'ReadOnlySequence<T>' is inaccessible due to its protection level
Library\PackageCache\com.neuecc.messagepack@7d06c13e72\MessagePackReader.cs(724,43): error CS0122: 'ReadOnlySpan<T>' is inaccessible due to its protection level
Library\PackageCache\com.neuecc.messagepack@7d06c13e72\MessagePackReader.cs(38,34): error CS0122: 'ReadOnlyMemory<T>' is inaccessible due to its protection level
Library\PackageCache\com.neuecc.messagepack@7d06c13e72\MessagePackReader.cs(49,37): error CS0122: 'ReadOnlySequence<T>' is inaccessible due to its protection level
Library\PackageCache\com.neuecc.messagepack@7d06c13e72\SequenceReader.cs(26,17): error CS0122: 'ReadOnlySequence<T>' is inaccessible due to its protection level
Library\PackageCache\com.neuecc.messagepack@7d06c13e72\SequenceReader.cs(31,17): error CS0122: 'SequencePosition' is inaccessible due to its protection level
Library\PackageCache\com.neuecc.messagepack@7d06c13e72\SequenceReader.cs(36,17): error CS0122: 'SequencePosition' is inaccessible due to its protection level
Library\PackageCache\com.neuecc.messagepack@7d06c13e72\SequenceReader.cs(41,17): error CS0122: 'ReadOnlyMemory<T>' is inaccessible due to its protection level
Library\PackageCache\com.neuecc.messagepack@7d06c13e72\MessagePackStreamReader.cs(50,16): error CS0122: 'ReadOnlySequence<T>' is inaccessible due to its protection level
Library\PackageCache\com.neuecc.messagepack@7d06c13e72\SequenceReader.cs(134,16): error CS0122: 'ReadOnlySpan<T>' is inaccessible due to its protection level
Library\PackageCache\com.neuecc.messagepack@7d06c13e72\MessagePackStreamReader.cs(70,57): error CS1983: The return type of an async method must be void, Task or Task<T>
Library\PackageCache\com.neuecc.messagepack@7d06c13e72\MessagePackStreamReader.cs(70,22): error CS0246: The type or namespace name 'ValueTask<>' could not be found (are you missing a using directive or an assembly reference?)
Library\PackageCache\com.neuecc.messagepack@7d06c13e72\MessagePackStreamReader.cs(70,32): error CS0122: 'ReadOnlySequence<T>' is inaccessible due to its protection level
Library\PackageCache\com.neuecc.messagepack@7d06c13e72\MessagePackStreamReader.cs(159,45): error CS0122: 'ReadOnlySequence<T>' is inaccessible due to its protection level
Library\PackageCache\com.neuecc.messagepack@7d06c13e72\MessagePackStreamReader.cs(25,17): error CS0122: 'SequencePosition' is inaccessible due to its protection level
Library\PackageCache\com.neuecc.messagepack@7d06c13e72\MessagePackWriter.cs(73,40): error CS0122: 'IBufferWriter<T>' is inaccessible due to its protection level
Library\PackageCache\com.neuecc.messagepack@7d06c13e72\MessagePackWriter.cs(98,30): error CS0122: 'ReadOnlySpan<T>' is inaccessible due to its protection level
Library\PackageCache\com.neuecc.messagepack@7d06c13e72\MessagePackWriter.cs(104,33): error CS0122: 'ReadOnlySequence<T>' is inaccessible due to its protection level
Library\PackageCache\com.neuecc.messagepack@7d06c13e72\MessagePackWriter.cs(742,27): error CS0122: 'ReadOnlySpan<T>' is inaccessible due to its protection level
Library\PackageCache\com.neuecc.messagepack@7d06c13e72\MessagePackWriter.cs(761,30): error CS0122: 'ReadOnlySequence<T>' is inaccessible due to its protection level
Library\PackageCache\com.neuecc.messagepack@7d06c13e72\MessagePackWriter.cs(944,34): error CS0122: 'ReadOnlySpan<T>' is inaccessible due to its protection level
Library\PackageCache\com.neuecc.messagepack@7d06c13e72\MessagePackWriter.cs(837,36): error CS0122: 'ReadOnlySequence<T>' is inaccessible due to its protection level
Library\PackageCache\com.neuecc.messagepack@7d06c13e72\MessagePackWriter.cs(854,33): error CS0122: 'ReadOnlySpan<T>' is inaccessible due to its protection level
Library\PackageCache\com.neuecc.messagepack@7d06c13e72\MessagePackWriter.cs(1067,16): error CS0122: 'Span<T>' is inaccessible due to its protection level
Library\PackageCache\com.neuecc.messagepack@7d06c13e72\MessagePackWriter.cs(1129,57): error CS0122: 'Span<T>' is inaccessible due to its protection level
Library\PackageCache\com.neuecc.messagepack@7d06c13e72\MessagePackWriter.cs(1131,55): error CS0122: 'Span<T>' is inaccessible due to its protection level
Library\PackageCache\com.neuecc.messagepack@7d06c13e72\MessagePackWriter.cs(1133,56): error CS0122: 'Span<T>' is inaccessible due to its protection level
Library\PackageCache\com.neuecc.messagepack@7d06c13e72\MessagePackWriter.cs(1135,58): error CS0122: 'Span<T>' is inaccessible due to its protection level
Library\PackageCache\com.neuecc.messagepack@7d06c13e72\MessagePackWriter.cs(1154,56): error CS0122: 'Span<T>' is inaccessible due to its protection level
Library\PackageCache\com.neuecc.messagepack@7d06c13e72\MessagePackWriter.cs(1177,57): error CS0122: 'Span<T>' is inaccessible due to its protection level
Library\PackageCache\com.neuecc.messagepack@7d06c13e72\MessagePackWriter.cs(1193,64): error CS0122: 'Span<T>' is inaccessible due to its protection level
Library\PackageCache\com.neuecc.messagepack@7d06c13e72\MessagePackWriter.cs(1195,65): error CS0122: 'Span<T>' is inaccessible due to its protection level
Library\PackageCache\com.neuecc.messagepack@7d06c13e72\MessagePackWriter.cs(38,34): error CS0122: 'IBufferWriter<T>' is inaccessible due to its protection level
Library\PackageCache\com.neuecc.messagepack@7d06c13e72\SequencePool.cs(47,26): error CS0122: 'ArrayPool<T>' is inaccessible due to its protection level
Library\PackageCache\com.neuecc.messagepack@7d06c13e72\SequenceReader.cs(108,16): error CS0122: 'ReadOnlySequence<T>' is inaccessible due to its protection level
Library\PackageCache\com.neuecc.messagepack@7d06c13e72\SequenceReader.cs(128,16): error CS0122: 'SequencePosition' is inaccessible due to its protection level
Library\PackageCache\com.neuecc.messagepack@7d06c13e72\SequenceReader.cs(144,16): error CS0122: 'ReadOnlySpan<T>' is inaccessible due to its protection level
Library\PackageCache\com.neuecc.messagepack@7d06c13e72\SequenceReader.cs(454,31): error CS0122: 'Span<T>' is inaccessible due to its protection level
Library\PackageCache\com.neuecc.messagepack@7d06c13e72\SequenceReader.cs(466,43): error CS0122: 'Span<T>' is inaccessible due to its protection level
Library\PackageCache\com.neuecc.messagepack@7d06c13e72\SequenceReader.cs(58,34): error CS0122: 'ReadOnlySequence<T>' is inaccessible due to its protection level
Library\PackageCache\com.neuecc.messagepack@7d06c13e72\SequenceReader.cs(85,31): error CS0122: 'ReadOnlyMemory<T>' is inaccessible due to its protection level
Library\PackageCache\com.neuecc.messagepack@7d06c13e72\StreamPolyfillExtensions.cs(25,54): error CS0122: 'Span<T>' is inaccessible due to its protection level
Library\PackageCache\com.neuecc.messagepack@7d06c13e72\StreamPolyfillExtensions.cs(55,70): error CS0122: 'Memory<T>' is inaccessible due to its protection level
Library\PackageCache\com.neuecc.messagepack@7d06c13e72\StreamPolyfillExtensions.cs(55,25): error CS0246: The type or namespace name 'ValueTask<>' could not be found (are you missing a using directive or an assembly reference?)
Library\PackageCache\com.neuecc.messagepack@7d06c13e72\StreamPolyfillExtensions.cs(92,56): error CS0122: 'ReadOnlySpan<T>' is inaccessible due to its protection level
Library\PackageCache\com.neuecc.messagepack@7d06c13e72\StreamPolyfillExtensions.cs(117,41): error CS1983: The return type of an async method must be void, Task or Task<T>
Library\PackageCache\com.neuecc.messagepack@7d06c13e72\StreamPolyfillExtensions.cs(117,72): error CS0122: 'ReadOnlyMemory<T>' is inaccessible due to its protection level
Library\PackageCache\com.neuecc.messagepack@7d06c13e72\StreamPolyfillExtensions.cs(117,31): error CS0246: The type or namespace name 'ValueTask' could not be found (are you missing a using directive or an assembly reference?)
Library\PackageCache\com.neuecc.messagepack@7d06c13e72\StringEncoding.cs(17,73): error CS0122: 'ReadOnlySpan<T>' is inaccessible due to its protection level
Library\PackageCache\com.neuecc.messagepack@7d06c13e72\Internal\Sequence`1.cs(113,41): error CS0122: 'ReadOnlySequence<T>' is inaccessible due to its protection level
Library\PackageCache\com.neuecc.messagepack@7d06c13e72\Internal\Sequence`1.cs(128,31): error CS0122: 'SequencePosition' is inaccessible due to its protection level
Library\PackageCache\com.neuecc.messagepack@7d06c13e72\Internal\Sequence`1.cs(184,16): error CS0122: 'Memory<T>' is inaccessible due to its protection level
Library\PackageCache\com.neuecc.messagepack@7d06c13e72\Internal\Sequence`1.cs(191,16): error CS0122: 'Span<T>' is inaccessible due to its protection level
Library\PackageCache\com.neuecc.messagepack@7d06c13e72\Internal\Sequence`1.cs(55,25): error CS0122: 'MemoryPool<T>' is inaccessible due to its protection level
Library\PackageCache\com.neuecc.messagepack@7d06c13e72\Internal\Sequence`1.cs(65,25): error CS0122: 'ArrayPool<T>' is inaccessible due to its protection level
Library\PackageCache\com.neuecc.messagepack@7d06c13e72\Internal\Sequence`1.cs(34,26): error CS0122: 'MemoryPool<T>' is inaccessible due to its protection level
Library\PackageCache\com.neuecc.messagepack@7d06c13e72\Internal\Sequence`1.cs(36,26): error CS0122: 'ArrayPool<T>' is inaccessible due to its protection level
Library\PackageCache\com.neuecc.messagepack@7d06c13e72\Unity\Extension\UnsafeBlitFormatter.cs(25,46): error CS0122: 'ReadOnlySpan<T>' is inaccessible due to its protection level
Library\PackageCache\com.neuecc.messagepack@7d06c13e72\Unity\Extension\UnsafeBlitFormatter.cs(25,70): error CS0122: 'Span<T>' is inaccessible due to its protection level
Library\PackageCache\com.neuecc.messagepack@7d06c13e72\Internal\Sequence`1.cs(301,41): error CS0122: 'ReadOnlySequenceSegment<T>' is inaccessible due to its protection level
Library\PackageCache\com.neuecc.messagepack@7d06c13e72\Internal\Sequence`1.cs(327,22): error CS0122: 'Memory<T>' is inaccessible due to its protection level
Library\PackageCache\com.neuecc.messagepack@7d06c13e72\Internal\Sequence`1.cs(332,22): error CS0122: 'Span<T>' is inaccessible due to its protection level
Library\PackageCache\com.neuecc.messagepack@7d06c13e72\Internal\Sequence`1.cs(338,22): error CS0122: 'IMemoryOwner<T>' is inaccessible due to its protection level
Library\PackageCache\com.neuecc.messagepack@7d06c13e72\Internal\Sequence`1.cs(343,22): error CS0122: 'Memory<T>' is inaccessible due to its protection level
Library\PackageCache\com.neuecc.messagepack@7d06c13e72\Internal\Sequence`1.cs(369,34): error CS0122: 'IMemoryOwner<T>' is inaccessible due to its protection level
Library\PackageCache\com.neuecc.messagepack@7d06c13e72\SafeBitConverter.cs(11,38): error CS0122: 'ReadOnlySpan<T>' is inaccessible due to its protection level
Library\PackageCache\com.neuecc.messagepack@7d06c13e72\Internal\Sequence`1.cs(388,39): error CS0122: 'ArrayPool<T>' is inaccessible due to its protection level
Library\PackageCache\com.neuecc.messagepack@7d06c13e72\Internal\Sequence`1.cs(359,42): warning CS0109: The member 'Sequence<T>.SequenceSegment.Next' does not hide an accessible member. The new keyword is not required.
Library\PackageCache\com.neuecc.messagepack@7d06c13e72\SafeBitConverter.cs(31,40): error CS0122: 'ReadOnlySpan<T>' is inaccessible due to its protection level
Library\PackageCache\com.neuecc.messagepack@7d06c13e72\SafeBitConverter.cs(33,41): error CS0122: 'ReadOnlySpan<T>' is inaccessible due to its protection level
Library\PackageCache\com.neuecc.messagepack@7d06c13e72\SafeBitConverter.cs(49,39): error CS0122: 'ReadOnlySpan<T>' is inaccessible due to its protection level
Library\PackageCache\com.neuecc.messagepack@7d06c13e72\MessagePackSerializer.Json.cs(55,44): error CS0122: 'ReadOnlyMemory<T>' is inaccessible due to its protection level
Library\PackageCache\com.neuecc.messagepack@7d06c13e72\MessagePackSerializer.Json.cs(61,47): error CS0122: 'ReadOnlySequence<T>' is inaccessible due to its protection level
Library\PackageCache\com.neuecc.messagepack@7d06c13e72\MessagePackSerializer.NonGeneric.cs(32,49): error CS0122: 'IBufferWriter<T>' is inaccessible due to its protection level
Library\PackageCache\com.neuecc.messagepack@7d06c13e72\MessagePackSerializer.cs(52,41): error CS0122: 'IBufferWriter<T>' is inaccessible due to its protection level
Library\PackageCache\com.neuecc.messagepack@7d06c13e72\MessagePackSerializer.NonGeneric.cs(74,53): error CS0122: 'ReadOnlyMemory<T>' is inaccessible due to its protection level
Library\PackageCache\com.neuecc.messagepack@7d06c13e72\MessagePackSerializer.NonGeneric.cs(80,53): error CS0122: 'ReadOnlySequence<T>' is inaccessible due to its protection level
Library\PackageCache\com.neuecc.messagepack@7d06c13e72\MessagePackSerializer.cs(200,43): error CS0122: 'ReadOnlySequence<T>' is inaccessible due to its protection level
Library\PackageCache\com.neuecc.messagepack@7d06c13e72\MessagePackSerializer.cs(259,40): error CS0122: 'ReadOnlyMemory<T>' is inaccessible due to its protection level
Library\PackageCache\com.neuecc.messagepack@7d06c13e72\MessagePackSerializer.cs(277,40): error CS0122: 'ReadOnlyMemory<T>' is inaccessible due to its protection level
Library\PackageCache\com.neuecc.messagepack@7d06c13e72\MessagePackSerializer.cs(289,40): error CS0122: 'ReadOnlyMemory<T>' is inaccessible due to its protection level
Library\PackageCache\com.neuecc.messagepack@7d06c13e72\MessagePackSerializer.NonGeneric.cs(68,23): error CS0246: The type or namespace name 'ValueTask<>' could not be found (are you missing a using directive or an assembly reference?)
Library\PackageCache\com.neuecc.messagepack@7d06c13e72\MessagePackSerializer.cs(363,42): error CS1983: The return type of an async method must be void, Task or Task<T>
Library\PackageCache\com.neuecc.messagepack@7d06c13e72\MessagePackSerializer.cs(363,29): error CS0246: The type or namespace name 'ValueTask<>' could not be found (are you missing a using directive or an assembly reference?)
Library\PackageCache\com.neuecc.messagepack@7d06c13e72\MessagePackSerializer.NonGeneric.cs(85,48): error CS1983: The return type of an async method must be void, Task or Task<T>
Library\PackageCache\com.neuecc.messagepack@7d06c13e72\MessagePackSerializer.NonGeneric.cs(85,30): error CS0246: The type or namespace name 'ValueTask<>' could not be found (are you missing a using directive or an assembly reference?)
Library\PackageCache\com.neuecc.messagepack@7d06c13e72\MessagePackSerializer.cs(414,139): error CS0122: 'ReadOnlySequence<T>' is inaccessible due to its protection level
Library\PackageCache\com.neuecc.messagepack@7d06c13e72\MessagePackSerializer.cs(444,44): error CS0122: 'ReadOnlySequence<T>' is inaccessible due to its protection level
Library\PackageCache\com.neuecc.messagepack@7d06c13e72\MessagePackSerializer.cs(444,74): error CS0122: 'Span<T>' is inaccessible due to its protection level
Library\PackageCache\com.neuecc.messagepack@7d06c13e72\MessagePackSerializer.cs(472,73): error CS0122: 'IBufferWriter<T>' is inaccessible due to its protection level
Library\PackageCache\com.neuecc.messagepack@7d06c13e72\Utilities.cs(36,64): error CS0122: 'IBufferWriter<T>' is inaccessible due to its protection level
Library\PackageCache\com.neuecc.messagepack@7d06c13e72\Utilities.cs(36,25): error CS0122: 'Memory<T>' is inaccessible due to its protection level
Library\PackageCache\com.neuecc.messagepack@7d06c13e72\MessagePackSerializer.cs(549,48): error CS0122: 'ReadOnlySequence<T>' is inaccessible due to its protection level
Library\PackageCache\com.neuecc.messagepack@7d06c13e72\MessagePackSerializer.cs(630,58): error CS0122: 'Span<T>' is inaccessible due to its protection level
Library\PackageCache\com.neuecc.messagepack@7d06c13e72\MessagePackSerializer.NonGeneric.cs(105,38): error CS0122: 'IBufferWriter<T>' is inaccessible due to its protection level
Library\PackageCache\com.neuecc.messagepack@7d06c13e72\MessagePackSerializer.NonGeneric.cs(109,93): error CS0246: The type or namespace name 'ValueTask<>' could not be found (are you missing a using directive or an assembly reference?)
Library\PackageCache\com.neuecc.messagepack@7d06c13e72\MessagePackSerializer.NonGeneric.cs(111,36): error CS0122: 'ReadOnlyMemory<T>' is inaccessible due to its protection level
Library\PackageCache\com.neuecc.messagepack@7d06c13e72\MessagePackSerializer.NonGeneric.cs(112,36): error CS0122: 'ReadOnlySequence<T>' is inaccessible due to its protection level
Library\PackageCache\com.neuecc.messagepack@7d06c13e72\MessagePackSerializer.cs(393,43): error CS0122: 'ReadOnlySpan<T>' is inaccessible due to its protection level
Library\PackageCache\com.neuecc.messagepack@7d06c13e72\MessagePackSerializer.cs(393,69): error CS0122: 'Span<T>' is inaccessible due to its protection level
```
#### Repro steps

Add the following to manifest.json
```json
"com.neuecc.messagepack": "https://github.com/neuecc/MessagePack-CSharp.git?path=src/MessagePack.UnityClient/Assets/Scripts/MessagePack#v2.1.165"
```
Remove "auto reference" from all required 

#### Expected behavior

Package imported withotu errors.

#### Actual behavior

Missing references to Unity 2020.1.2f1

- Version used: 2.1.165
- Runtime: Unity 2020.1.2f1

#### Additional context

The package.json expects all dlls to be auto-referenced. This is a horrible thing Unity does to not have to specify your dependencies. We do know all dependencies we need and can references them.

--- a/src/MessagePack.UnityClient/Assets/Scripts/MessagePack/MessagePack.asmdef
+++ b/src/MessagePack.UnityClient/Assets/Scripts/MessagePack/MessagePack.asmdef
@@ -5,8 +5,14 @@
     "includePlatforms": [],
     "excludePlatforms": [],
     "allowUnsafeCode": true,
-    "overrideReferences": false,
-    "precompiledReferences": [],
+    "overrideReferences": true,
+    "precompiledReferences": [
+      "System.Memory.dll"
+      ,"System.Buffers.dll"
+      ,"System.Threading.Tasks.Extensions.dll"
+      ,"System.Runtime.CompilerServices.Unsafe.dll"
+      ,"System.Runtime.Extensions.dll"
+    ],
     "autoReferenced": true,
     "defineConstraints": [],
     "versionDefines": []

